test/OptionSettlementEngine.t.sol

```diff
@@ -1830,14 +1842,20 @@ contract OptionSettlementTest is Test, NFTreceiver {
     //                            TEST HELPERS
     // **********************************************************************
```

Comment:
If you're going HAM on test layout soon, update the section separator styling to match the production code.

Reply:
Also, bump on the ASCII art bc it seems you're in that mood. Here is some ideas I pulled together, again using these 2 sites:

https://github.com/valorem-labs-inc/valorem-core/blob/docs/ascii-art/src/OptionSettlementEngine.sol

- Image — https://www.ascii-art-generator.org/
- Text — https://textkool.com/en/ascii-art-generator?hl=default&vl=default&font=Red%20Phoenix&text=Your%20text%20here%20

Reply:
Let me ask the designer.